### PR TITLE
Remove instructions to enable unsecure builds.

### PR DESCRIPTION
**Pull request by SeppoTakalo**
Add link to mbed TLS guide instead.

### CODE REVIEW

**Comment by SeppoTakalo** [comment]
@kjbracey-arm @sbutcher-arm Would this be better that directly instructing users to turn on unsecure options?
If so, I would send same pr for client example also.

**Comment by SeppoTakalo** [comment]
@peknis01 Please  review.

File: README.md [hunk 1]
```diff
@@ -74,15 +74,12 @@ The following tables show the values you should use in the `mbed_app.json` file
 
 **Note:** You need to recompile your application after modifying the configurations by issuing the command `mbed compile -c`.
 
-### Requirements for boards without entropy
+### Requirements for hardware
 
-To run this example application there must be a source of entropy. If the platform you are using does not have
-an entropy source you need edit the `mbed_app.json` file to add `NULL_ENTROPY` feature for mbedTLS:
+The networking stack used in this example requires TLS functionality to be enabled on mbed TLS.
```
Comment (peknis01) [inline README.md]: Do you need to say that TLS functionality needs to be enabled on mbed TLS? Would it be enough to say that TLS functionality must be enabled?

**Comment by sbutcher-arm** [comment]
LGTM.

**Pull request merged**